README.md

```diff
@@ -330,120 +477,69 @@ Additionally, you can also locally trigger a deep link to make sure your code is
-
-- Once you've added in the dependency make sure to `pod install`.
-- Then in the `NotificationService.swift` file add the code for the two required delegates from [this](Examples/KlaviyoSwiftExamples/CocoapodsExample/NotificationServiceExtension/NotificationService.swift) file. This sample covers calling into Klaviyo so that we can download and attach the media to the push notification.
-
-### Step 3: Test your rich push notifications
-
-#### Local testing
-
-There are three things you would need to do this -
-
-1. Any push notifications tester such as [this](https://github.com/onmyway133/PushNotifications).
-2. A push notification payload that resembles what Klaviyo would send to you. The below payload should work as long as the image is valid:
-
-```json
-{
-  "aps": {
-    "alert": {
-      "title": "Free apple vision pro",
-      "body": "Free Apple vision pro when you buy a Klaviyo subscription."
-    },
-    "mutable-content": 1
-  },
-  "rich-media": "https://www.apple.com/v/apple-vision-pro/a/images/overview/hero/portrait_base__bwsgtdddcl7m_large.jpg",
-  "rich-media-type": "jpg"
+```swift
+extension AppDelegate: UNUserNotificationCenterDelegate {
+    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+        let handled = KlaviyoSDK().handle(notificationResponse: response, withCompletionHandler: completionHandler) { url in
+            print("deep link is ", url)
+        }
+        if !handled {
+           // not a klaviyo notification should be handled by other app code
+        }
+    }
 }
 ```
 
-3. A real device's push notification token. This can be printed out to the console from the `didRegisterForRemoteNotificationsWithDeviceToken` method in `AppDelegate`.
-
-Once we have these three things we can then use the push notifications tester and send a local push notification to make sure that everything was set up correctly.
-
+Note that the deep link handler will be called back on the main thread. If you want to handle URI schemes in addition to universal links you implement them as described in [Option1: URI Schemes](#option-1-uri-schemes).
 
-#### Sandbox Support
+## Additional Details
 
-Apple has two environments with push notification support - production and Sandbox. The Production environment supports sending push notifications to real users when an app is published in the App Store or TestFlight. In contrast, Sandbox applications that support push notifications are those signed with iOS Development Certificates, instead of iOS Distribution Certificates. Sandbox acts as a staging environment, allowing you to test your applications in a environment similar to but distinct from production without having to worry about sending messages to real users.
+### Sandbox Support
+Apple has two environments with push notification support - production and Sandbox.
```

Comment:
Production should be capitalized to be consistent

Reply:
Done.